scripts/process_wang.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+. ./scripts/utils.sh
+
+set -o errexit
+
+main() {
+    # Download and process data which describes cell locations, boundaries,
+    # and gene expression levels. Multiple JSON output files are produced:
+    # The files are redudant, but this reduces the processing that needs
+    # to be done on the client-side.
+
+    CSV_IN="$INPUT/mermaid.csv"
+    PNG_IN="$INPUT/mermaid.png"
+
+    CLI_ARGS="--csv_file $CSV_IN"
+    add_arg 'cells' 'mermaid'
+    add_arg 'molecules' 'mermaid'
+    add_arg 'images' 'mermaid'
+
+    echo "Download and process cells..."
+
+    [ -e "$CSV_IN" ] || \
+    (wget "$MERMAID_URL/data.csv.gz" -O "$CSV_IN.gz" && gunzip -df "$CSV_IN")
```

Comment:
I would keep this as an if-then:
- Long chains of mixed operations get confusing.
- In particular, `||` and `&&` are of equal precedence in bash, so it's easy to trip up.
- `()` isn't just for grouping: It's actually a [subshell](http://www.tldp.org/LDP/abs/html/subshells.html) ... so there's the cost of starting a new process, and variable scope might not be what you expect, etc.